[2025-04-17T21:30:45.675+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-17T21:30:45.681+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: triphobo_attractions_pipeline.upload_raw_attractions manual__2025-04-17T20:27:09.818831+00:00 [queued]>
[2025-04-17T21:30:45.683+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: triphobo_attractions_pipeline.upload_raw_attractions manual__2025-04-17T20:27:09.818831+00:00 [queued]>
[2025-04-17T21:30:45.683+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-04-17T21:30:45.687+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upload_raw_attractions> on 2025-04-17 20:27:09.818831+00:00
[2025-04-17T21:30:45.690+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1472) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-17T21:30:45.693+0000] {standard_task_runner.py:72} INFO - Started process 1473 to run task
[2025-04-17T21:30:45.692+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'triphobo_attractions_pipeline', 'upload_raw_attractions', 'manual__2025-04-17T20:27:09.818831+00:00', '--job-id', '409', '--raw', '--subdir', 'DAGS_FOLDER/attractions_dag.py', '--cfg-path', '/tmp/tmp0hgjp1e4']
[2025-04-17T21:30:45.697+0000] {standard_task_runner.py:105} INFO - Job 409: Subtask upload_raw_attractions
[2025-04-17T21:30:45.731+0000] {task_command.py:467} INFO - Running <TaskInstance: triphobo_attractions_pipeline.upload_raw_attractions manual__2025-04-17T20:27:09.818831+00:00 [running]> on host 4d0efc92e922
[2025-04-17T21:30:45.764+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='triphobo_attractions_pipeline' AIRFLOW_CTX_TASK_ID='upload_raw_attractions' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T20:27:09.818831+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-17T20:27:09.818831+00:00'
[2025-04-17T21:30:45.765+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-17T21:30:47.856+0000] {upload_to_S3.py:33} ERROR - Failed to upload /tmp/triphobo_data/multi_city_attractions.csv to s3://None/raw/multi_city_attractions.csv: expected string or bytes-like object, got 'NoneType'
[2025-04-17T21:30:47.857+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-17T21:30:47.862+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-17T21:30:47.862+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=triphobo_attractions_pipeline, task_id=upload_raw_attractions, run_id=manual__2025-04-17T20:27:09.818831+00:00, execution_date=20250417T202709, start_date=20250417T213045, end_date=20250417T213047
[2025-04-17T21:30:47.890+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-17T21:30:47.902+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-17T21:30:47.902+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
